### ISSUE TITLE
Simplifying setup

### ISSUE THREAD

**Issue by Avamander** [mumble-voip/mumble#3260]
I try to quite heavily promote people to start using Mumble for it's superior performance and usability, but the issue is that people get stuck in the setup far too often. 

So my suggestion would be somehow simplifying the setup somehow, any ideas how to do so exactly are really welcome. I thought about having a simple setup by default that auto-detects where the mic is plugged in, sets the audio bandwith automatically, skips the positional audio question and then just lets a person set a push-to-talk key. If someone wishes to use auto-speak functionality they can use the advanced setup later (that also reminded me that push-to-talk key can't be changed from the settings, only Audio wizard, which would be a nice thing to fix too).

**Comment by reelsense** [comment]
I have similar experiences. Every time a first-time user ends up sitting in a channel for 10 min while 1 or 2 people try to explain what to do, while the user can't respond.

**Comment by preterive** [comment]
I have similar experiences.

**Comment by EmperorArthur** [comment]
I agree with everything but the ptt default.    I'd argue that speech detection, with automatic gain control is far more common.  For better or worse, that's what Discord uses, and it's become the de-facto "just works" standard.

**Comment by reelsense** [comment]
@EmperorArthur I'm ok with that default as long as I can _force_ and/or _enforce_ PTT at the server or channel level.

**Comment by Avamander** [comment]
@EmperorArthur SD is common with auto-gain, but currently Mumble has no auto-gain, thus it's hard to set up for far too many. If mumble had that feature I would have suggested it to be the default and skip that step too.

**Comment by reelsense** [comment]
Mumble has AGC and yes PTT can be setup in settings, but it's not at all intuitive. There are too many options. PTT options should be reduced to a [_bind_] button, and everything else should be hidden behind the _advanced_ button.

**Comment by Avamander** [comment]
> yes PTT can be setup in settings, but it's not at all intuitive. There are too many options.

It's really not where it's supposed to be if what you say is true.

> Mumble has AGC

Where?

**Comment by reelsense** [comment]
> It's really not where it's supposed to be if what you say is true

Shortcuts.

>Where?

It's always on and always has been. https://wiki.mumble.info/wiki/Features

**Comment by Avamander** [comment]
> It's always on and always has been. https://wiki.mumble.info/wiki/Features

Not the same as Discord's though, which is what EmperorArthur meant, it's not sufficient to skip manual level-to-transmit calibration.

**Comment by reelsense** [comment]
Maybe Discord (or Skype or any of those products) is licensing some library that levels automatically without calibration. I bet Mumble could pick a safe default and just prompt to calibrate when it detects clipping or -40db audio.
